lib/v2/datasource/remote/model/invite_model.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:equatable/equatable.dart';
+
+class InviteModel extends Equatable {
+  final int? inviteId;
+  final String? transferQuantity;
+  final String? sowQuantity;
+  final String? sponsor;
+  final String? account;
+  final String? inviteHash;
+  final String? inviteSecret;
+
+  const InviteModel({
+    this.inviteId,
+    this.transferQuantity,
+    this.sowQuantity,
+    this.sponsor,
+    this.account,
+    this.inviteHash,
+    this.inviteSecret,
+  });
+
+  factory InviteModel.fromJson(Map<String, dynamic> json) {
+    return InviteModel(
+      inviteId: json['invite_id'],
+      transferQuantity: json['transfer_quantity'],
+      sowQuantity: json['sow_quantity'],
+      sponsor: json['sponsor'],
+      account: json['account'],
+      inviteHash: json['invite_hash'],
+      inviteSecret: json['invite_secret'],
+    );
+  }
+
+  @override
+  List<Object?> get props => [
+        inviteId,
+        transferQuantity,
+        sowQuantity,
+        sponsor,
+        account,
+        inviteHash,
+        inviteSecret,
+      ];
+
+  @override
+  bool operator ==(Object other) {
```

Comment:
General opinion on equality here - in my experience it's best to explicitly test equality and not rely on language features, because that way I can control exactly what is being compared. 

The gain from using the generic features like "hey I can just use == and it works" is comparatively minor to the potential downside of forgetting to exclude a new variable that's added a year later, and then thinking about how that affects the '=='. 

The downside is unexpected effects that no one thought about when adding or removing fields from the object. 

It's like JavaScript - works at first, problems later ;)

Reply:
Well first of all I think that many of these api models are not used to make comparison between them, in the case of balanceModel and planted model there are no comparisons of these instances as far as I have worked.

Regarding use Equatable, just don't forget to put the new field in the props array ;)